session/socketforward/socket.go

```diff
@@ -0,0 +1,109 @@
+package socketforward
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/moby/buildkit/session"
+	"github.com/pkg/errors"
+	context "golang.org/x/net/context"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc/metadata"
+)
+
+const SocketID = "socket_id"
```

Comment:
The intention is slightly clearer

```suggestion
const SocketIDKey = "socket_id"
```